src/experiments.js

```diff
@@ -104,24 +104,31 @@ function calcExperimentOn(win, experimentId) {
 
 
 /**
- * Toggles the expriment on or off. Returns the actual value of the expriment
+ * Toggles the experiment on or off. Returns the actual value of the experiment
  * after toggling is done.
  * @param {!Window} win
  * @param {string} experimentId
  * @param {boolean=} opt_on
+ * @param {boolean=} opt_saveExperimentIds  Whether to save the experiment IDs
+ *     to the cookie after toggling or not.
  * @return {boolean}
  */
-export function toggleExperiment(win, experimentId, opt_on) {
+export function toggleExperiment(win, experimentId, opt_on,
+    opt_saveExperimentIds) {
   const experimentIds = getExperimentIds(win);
   const currentlyOn = experimentIds.indexOf(experimentId) != -1;
   const on = opt_on !== undefined ? opt_on : !currentlyOn;
   if (on != currentlyOn) {
     if (on) {
       experimentIds.push(experimentId);
+      EXPERIMENT_TOGGLES[experimentId] = true;
     } else {
       experimentIds.splice(experimentIds.indexOf(experimentId), 1);
+      EXPERIMENT_TOGGLES[experimentId] = false;
+    }
+    if (opt_saveExperimentIds === undefined || opt_saveExperimentIds) {
```

Comment:
Done.  Renamed, which flipped the sense of the variable but made internal logic clearer.